### PR TITLE
12189 open dsa issues

### PR DESCRIPTION
**Pull request by sergei-bronnikov**
https://codio.myjetbrains.com/youtrack/issue/codio-12189

### CODE REVIEW

File: converter/rst/character.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class Character(object):
+    def __init__(self, source_string):
+        self.str = source_string
+        self._character_re = re.compile(r"""\\\\\$(?=\d)""")
+
+    @staticmethod
+    def _character(matchobj):
+        return "<span>$</span>"
```
Comment (destitutus) [inline converter/rst/character.py:11]: use single quotes

File: converter/rst/definition.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    def _definition(self, matchobj):
+        caret_token = self._caret_token
+        term = matchobj.group('term')
+        term = term.strip('**').strip()
+        content = matchobj.group('content')
+
+        space = re.search('\n *', content)
+        space_count = len(space.group(0)) - 1
+        space_regex = f"\n^ {{{space_count}}}"
```
Comment (destitutus) [inline converter/rst/definition.py:21]: use single quotes

File: converter/rst/inlineav.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+from string import Template
+from collections import namedtuple
+from converter.rst.utils import css_property
+
+IframeImage = namedtuple('IframeImage', ['src', 'path', 'content'])
+GUIDES_CDN = '//static-assets.codio.com/guides/opendsa/v1'
+MATHJAX_CDN = '//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1'
+JSAV_IFRAME_SUBPATH = 'jsav/iframe/v1/'
+JSAV_IMAGE_IFRAME = f"""
+<html>
```
Comment (destitutus) [inline converter/rst/inlineav.py:12]: see the comment about templates

File: converter/rst/list.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import re
+
+FOUR_SPACES = '    '
```
Comment (destitutus) [inline converter/rst/list.py:3]: `FOUR_SPACES = '    '` -> `TAB_SIZE = 4`

File: converter/rst/list.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        marker = matchobj.group('marker')
+        indent_size = 0
+        if tabs:
+            indent_size = len(tabs)
+        if marker and marker == "|":
+            indent_size += 1
+        if indent_size >= 1:
+            indent_size = indent_size - 1
+        content = '* '
+        content = indent_size * FOUR_SPACES + content
```
Comment (destitutus) [inline converter/rst/list.py:23]: `content = ' ' * indent_size * TAB_SIZE + content`
also, is it expected - if `indent_size == 0` - no indents?

File: tests/unit/cases_rst/only.rst [hunk 6]
```diff
@@ -0,0 +1,6 @@
+.. only:: CS2
+
+   Tight Bounds
+   ~~~~~~~~~~~~
+
+   The definition for big-Oh..
```
Comment (destitutus) [inline tests/unit/cases_rst/only.rst:6]: the result file is empty, is it ok?

File: converter/rst/list.py (hunk 5 above)
Comment (sergei-bronnikov) [reply converter/rst/list.py:23]: yes

File: tests/unit/cases_rst/only.rst (hunk 6 above)
Comment (sergei-bronnikov) [reply tests/unit/cases_rst/only.rst:6]: its OK. It serves to limit content by audience. The html version is also missing

**Review by destitutus — APPROVED** [review]

**Pull request merged**